tests/test_bgp_neighbor_af.rb

```diff
@@ -357,14 +491,28 @@ def default_originate(af, dbg)
                  'from false state')
 
     # Test default route-map, from true
-    af.default_originate_set(true, af.default_default_originate_route_map)
-    refute(af.default_originate_route_map,
-           "Test 5. #{dbg} Failed to set default route-map from existing")
+    if platform == :ios_xr
+      # XR does not support a default route-policy
+      assert_raises(ArgumentError) do
+        af.default_originate_set(true, af.default_default_originate_route_map)
+      end
+    else
+      af.default_originate_set(true, af.default_default_originate_route_map)
+      refute(af.default_originate_route_map,
+             "Test 5. #{dbg} Failed to set default route-map from existing")
+    end
 
     # Test default_state
-    af.default_originate_set(af.default_default_originate)
-    refute(af.default_originate,
-           "Test 6. #{dbg} Failed to set state to default")
+    if platform == :ios_xr
+      # XR does not support a default route-policy
+      assert_raises(ArgumentError) do
+        af.default_originate_set(af.default_default_originate)
```

Comment:
Well I looked up the command and see that I am completely wrong about the functionality. You CAN do default-originate without a policy-map, which means 'no should work. So reworking this code. I think my getter was wrong and gave me the wrong impression that it wasn't supported.